examples/experimental/light-js/index.html

```diff
@@ -76,11 +76,11 @@ <h2>Store</h2>
         createDecoder,
         utf8ToBytes,
         bytesToUtf8,
-      } from "https://unpkg.com/@waku/sdk@0.0.24-070b625.0/bundle/index.js";
+      } from "https://unpkg.com/@waku/sdk@0.0.24/bundle/index.js";
       import {
         enrTree,
         DnsNodeDiscovery,
-      } from "https://unpkg.com/@waku/discovery@0.0.2-070b625.0/bundle/index.js";
```

Comment:
is it intentional? I think latest is better to be used

Reply:
updated to use current next tag